src/main/scala/org/scalacheck/Prop.scala

```diff
@@ -765,8 +766,8 @@ object Prop {
     /*
      * Returns the first failed result in Left or success in Right.
      */
-    def getFirstFailure(xs: Stream[T]): Either[(T,Result),(T,Result)] = {
-      assert(!xs.isEmpty, "Stream cannot be empty")
+    def getFirstFailure(xs: LazyList[T]): Either[(T,Result),(T,Result)] = {
```

Comment:
Is this binary compatible?

Reply:
Yes, because `LazyList` is an alias for `Stream` when building on 2.10-2.12 (https://github.com/rickynils/scalacheck/blob/543b4056bffdb684faec49e8afb09aee4c24eb2f/src/main/scala-2.13-/org/scalacheck/ScalaVersionSpecific.scala#L19-L20)

Reply:
:+1: